jekyll/_includes/snippets/gpu-linux-resource-table.md

```diff
@@ -1,5 +1,9 @@
 Class | vCPUs | RAM | GPUs | GPU model | GPU Memory (GiB) | Disk Size (GiB) | Cloud | Server
 ---|---|---|---|---|---|---|---|---
+gpu.nvidia.small |   4   | 16  | 1 | Nvidia Tesla P4 | 16 | 300 | <i class="fa fa-check" aria-hidden="true"></i> | <i class="fa fa-times" aria-hidden="true"></i>
+gpu.nvidia.small.gen2 |   4   | 16  | 1 | Nvidia A10G | 16 | 300 | <i class="fa fa-check" aria-hidden="true"></i> | <i class="fa fa-times" aria-hidden="true"></i>
+gpu.nvidia.small.multi |   4   | 15  | 2 | Nvidia Tesla T4 | 16 | 300 | <i class="fa fa-check" aria-hidden="true"></i> | <i class="fa fa-times" aria-hidden="true"></i>
+gpu.nvidia.medium.multi |   8   | 30  | 4 | Nvidia Tesla T4 | 16 | 300 | <i class="fa fa-check" aria-hidden="true"></i> | <i class="fa fa-times" aria-hidden="true"></i>
 gpu.nvidia.medium |   8   | 30  | 1 | Nvidia Tesla T4 | 16 | 300 | <i class="fa fa-check" aria-hidden="true"></i> | <i class="fa fa-times" aria-hidden="true"></i>
```

Comment:
think the disc size for all gpus right now is 150 -> do we need to update these?

Reply:
can you confirm it's 150? if yes, i'll update

Reply:
Default is 150: https://github.com/circleci/machine-provisioner/blob/c365d9101509a29fa7dc27e7dbcd7683313da31a/provider/provider.go#L12

We dont specify an override here like we do for windows disk stuff: https://github.com/circleci/machine-provisioner/blob/main/deploy/conf/machine-provisioner-config.yml#L920